internal/provider/resource_gitlab_project_milestone_test.go

```diff
@@ -0,0 +1,96 @@
+package provider
```

Comment:
```suggestion
//go:build acceptance
// +build acceptance

package provider
```

required after the rebase onto `main`, see #1045